packages/perseus/src/widgets/__tests__/radio_test.jsx

```diff
@@ -347,8 +336,12 @@ describe("single-choice question", () => {
                 ).toBeVisible();
             });
 
-            it("should open the cross-out menu when button clicked", () => {
+            it("should open the cross-out menu when button clicked", async () => {
                 // Arrange
+
+                // Popper isn't a super well behaved component and warns that
+                // the component must be wrapped in act()
+                jest.spyOn(console, "error").mockImplementation(() => {});
```

Comment:
I think a better approach would be to fix these underlying errors that React is complaining about like this:
```suggestion
                await act(async () => {
                    await jest.runAllTimers();
                });
```

See https://reactjs.org/docs/testing-recipes.html#act